src/pudl/extract/xbrl.py

```diff
@@ -120,8 +125,8 @@ def convert_form(
     # we already delete the existing base on `clobber=True` in `xbrl2sqlite`
     run_main(
         filings=filings_archives,
-        db_path=sql_path,
-        clobber=False,
+        sqlite_path=sqlite_path,
+        duckdb_path=duckdb_path,
```

Comment:
did you mean to remove `clobber=False` here?

Reply:
Yes, the `clobber` option has been removed. It always clobbers now, since we never want to layer multiple extractions on top of each other in the same DB.

Reply:
ah okay i got confused but i didn't realize this `run_main` is not in this repo and i didn't see that change.